docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
+  - [Adding or deleting a remark: ](#adding-or-deleting-a-remark-remark) `remark`
+  - [Upload a student's photo: ](#uploading-a-students-photo-upload) `upload`
+  - [Display a summary of a student's attendance: ](#viewing-a-students-attendance-throughout-the-course-attendance) `attendance`
+- [Managing Sessions: ](#managing-sessions)
+  - [Adding a session: ](#adding-a-session-add-session) `add session`
+  - [Deleting a session: ](#deleting-a-session-delete-session) `delete session`
+  - [Editing a session: ](#editing-a-session-edit-session) `edit session`
+  - [Selecting a session to start attendance-taking: ](#selecting-a-session-to-start-attendance-taking-select) `select`
+  - [Finding a session: ](#finding-a-session-find-session) `find session`
+- [Managing Attendance: ](#attendance-page-commands)
+  - [Marking attendance of a selected students: ](#marking-selected-students-as-present-mark) `mark`
+  - [Unmarking attendance of a selected students: ](#marking-selected-students-as-absent-unmark) `unmark`
+  - [View the overall attendance of a session: ](#view-the-overall-attendance-of-a-session-attendance) `attendance`
+- [Managing Tasks: ](#managing-tasks)
+  - [Adding a task: ](#adding-a-task-add-task) `add task`
+  - [Deleting a task: ](#deleting-a-task-delete-task) `delete task`
+  - [Editing a task: ](#editing-a-task-edit-task) `edit task`
+  - [Selecting a task to assign grades: ](#selecting-a-task-to-assign-grades-select) `select`
+  - [Finding a task: ](#finding-a-task-find-task) `find task`
+- [Managing Grades: ](#managing-the-grades-of-a-task)
+  - [Assigning a grade: ](#assigning-a-grade-to-a-student-assign) `assign`
+
+### Commands
+
+Features in CLIpboard are supported by commands.
+Commands are like giving instructions to CLIpboard on what to do.
+
+A typical command would look like this:
+<br>
+<img src="./images/typicalcommand.png" width="400" height="138"/>
+
+**Command keyword(s)**: Typically 1 or 2 words, they specify the type of action for CLIpboard to execute
+**Supplementary command information**: These provide additional context about a command you wish to execute. Conventionally, these are referred to as "arguments" or "parameters".
+
+Putting it all together for the above example, `find course` would tell CLIpboard that you're looking for a course, and "cs2101" tells CLIpboard which specific course you're looking for.
+
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:** <br>
 
+* If a command expects a parameter to be input only once, but multiple instances of it are included, only the last occurrence of the parameter will be taken.<br>
+  * e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+* Extraneous parameters for commands that do not expect any parameters will be ignored.<br>
+  * e.g. if you input the command `help 123`, CLIpboard will interpret it simply as `help`.
+
+</div>
```

Comment:
Would it be better for this to be a `Tip` instead of a `Caution`?

Reply:
i've defined tip to be specifically about improving a user's skills. And caution is defined as critical info abt clipboard behaviour. i think under this definition would fit better as caution